.storybook/blocks/ComponentHeader/ComponentHeader.tsx

```diff
@@ -33,7 +33,7 @@ export const ComponentHeader: React.FC<ComponentHeaderProps> = ({
     <div className={styles.component}>
       <Title>{name}</Title>
       <div className={styles.version}>{version}</div>
-      {stage && <Status stage={stage} />}
+      {stage != null && stage !== undefined && <Status stage={stage} />}
```

Comment:
I guess too many checks.
Could you consider `isInteger` or set `default value` fix ways?